Makefile

```diff
@@ -62,9 +62,12 @@ generate::
 	$(call STEP_MESSAGE)
 	echo "This command does not do anything anymore. It will be removed in a future version."
 
-build:: build-proto go.ensure
+build:: build-proto build_wasm_display go.ensure
 	cd pkg && go install -ldflags "-X github.com/pulumi/pulumi/pkg/v3/version.Version=${VERSION}" ${PROJECT}
 
+build_wasm_display:: go.ensure
```

Comment:
I don't think `make build` is being run in CI anywhere other than in a cronjob.  We should probably run this on every PR as well.

Reply:
definitely

Reply:
Ah interesting--I assumed that this was being called during PRs. Agree that this should be run on each PR. I'll make the change.

Reply:
Okay, so if I'm reading this right, we always use Goreleaser to build binaries in CI?

Reply:
nvm, got it